src/components/ResponsiveAppBar.jsx

```diff
@@ -156,12 +156,9 @@ const ResponsiveAppBar = () => {
               component="a"
               href="/"
               sx={{
-                mr: 2,
-                fontFamily: "monospace",
-                fontWeight: 700,
-                letterSpacing: ".3rem",
-                color: "inherit",
-                textDecoration: "none",
+                display: { xs: "none", md: "flex" },
+                flexDirection: "row",
+                alignItems: "baseline",
               }}
```

Comment:
Why? I mean you're already making a big PR, if you spot some styling improvement, you can put them in a dedicated PR